src/components/SkeletonBodyText/SkeletonBodyText.tsx

```diff
@@ -17,7 +17,5 @@ export function SkeletonBodyText({lines = 3}: SkeletonBodyTextProps) {
     bodyTextLines.push(<div className={styles.SkeletonBodyText} key={i} />);
   }
 
-  return (
-    <div className={styles.SkeletonBodyTextContainer}>{bodyTextLines}</div>
-  );
+  return <>{bodyTextLines}</>;
```

Comment:
This doesn't need to be in a fragment it looks like
```suggestion
  return bodyTextLines;
```

Reply:
Let me double check. I had a type error without it 👀